### PR TITLE
#600: remove img style to prevent img downscaling

### LINKED ISSUE #250: Figure Size
Thanks for the awesome package! 

Would it be possible to make the figure size adjustable? I've tried to adjust it using matplotlib, but didn't seem work.

### LINKED ISSUE #600: How is the displayed figure size set in Atom/Hydrogen?
I'm not sure this is the right place to ask this question, but here goes.

I can't figure out how the displayed size of a pyplot figure in Hydrogen is determined. It seems like there is a maximum pixel area or something. Here are two example screenshots. Note that the widths of the two figures should be the same, but if I increase the height, the overall width is reduced (compare figure widths to the 80 character text width line). It's harder to tell, but the second figure is slightly taller than the first. What's going on? The overarching goal here is to get Atom/Hydrogen to display larger figures.

**figsize=(12,8)**
screen shot 2017-01-27 at 17 42 46

**figsize=(12,10)**
screen shot 2017-01-27 at 17 43 09

### PR DESCRIPTION
**Pull request by Earthson**
EDIT:
Fixes #250 
Fixes #600

### CODE REVIEW

File: styles/hydrogen.less [hunk 1]
```diff
@@ -398,9 +394,6 @@
             }
 
             video, img, svg {
```
Comment (lgeiger) [inline styles/hydrogen.less]: Since it is empty now we can remove the whole `video, img, svg` selector.

**Comment by lgeiger** [comment]
I really like the new behavior.
@n-riesco @nikitakit @rgbkrk What do you think?
scale

**Comment by n-riesco** [comment]
I also prefer the new behaviour, but not in the watch view.

**Comment by lgeiger** [comment]
> I also prefer the new behaviour, but not in the watch view.

Good point! The watch view should still scale to fit.

**Comment by Earthson** [comment]
@lgeiger Is it possible to make bubble max-height configurable?

**Comment by lgeiger** [comment]
Sorry closed it by accident. -> #609

**Pull request closed without merging**